hikari/api/rest.py

```diff
@@ -1201,6 +1202,12 @@ async def create_message(
             reader using Discord's TTS (text-to-speech) system.
         reply : hikari.undefined.UndefinedOr[hikari.snowflakes.SnowflakeishOr[hikari.messages.PartialMessage]]
             If provided, the message to reply to.
+        reply_fail_if_not_exists : hikari.undefined.UndefinedOr[bool]
```

Comment:
did you consider a less verbose name like `reply_must_exist`?

Reply:
This one is even better, didn't think about it. Ty!

Reply:
Renamed :)